assets/javascripts/discourse-sift/lib/report-reason.js.es6

```diff
@@ -0,0 +1,22 @@
+import showModal from "discourse/lib/show-modal";
+import loadScript from "discourse/lib/load-script";
+
+export function showReportReason(reviewable, callback, opts) {
+  opts = opts || {};
```

Comment:
opts are never used

Reply:
No, ended up not needing opts to pass anything, but it looked like a best practice especially if it is needed in the future